src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -0,0 +1,289 @@
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        uint amount =
+            IBondingCurveBase_v1(fundingManager).projectCollateralFeeCollected();
+        IBondingCurveBase_v1(fundingManager).withdrawProjectCollateralFee(
+            to, amount
+        );
+        emit IPIM_WorkflowFactory_v1.PimFeeClaimed(_msgSender(), amount);
+    }
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function transferPimFeeEligibility(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        _pimFeeRecipients[fundingManager] = to;
+        emit IPIM_WorkflowFactory_v1.PimFeeRecipientUpdated(_msgSender(), to);
+    }
+
+    //--------------------------------------------------------------------------
+    // Internal Functions
+
+    function _manageInitialSupplies(
+        IBondingCurveBase_v1 fundingManager,
+        IERC20 collateralToken,
+        ERC20Issuance_v1 issuanceToken,
+        uint initialCollateralSupply,
+        uint initialIssuanceSupply,
+        address recipient,
+        bool withInitialLiquidity
+    ) private {
+        if (withInitialLiquidity) {
+            // collateral token is paid for by the msg.sender
+            collateralToken.transferFrom(
+                _msgSender(), address(fundingManager), initialCollateralSupply
+            );
+            // issuance token is minted to the the specified recipient
+            issuanceToken.mint(recipient, initialIssuanceSupply);
+        } else {
+            // issuance token is minted to the burn address
+            issuanceToken.mint(address(0xDEAD), initialIssuanceSupply);
+        }
+    }
+
+    function _manageInitialPurchase(
+        IBondingCurveBase_v1 fundingManager,
+        IERC20 collateralToken,
+        uint firstCollateralIn,
+        address recipient
+    ) private {
+        // transfer initial collateral amount from deployer to factory
+        collateralToken.transferFrom(
+            _msgSender(), address(this), firstCollateralIn
+        );
+
+        // set allowance for curve to spend factory's tokens
+        collateralToken.approve(address(fundingManager), firstCollateralIn);
+
+        // make first purchase
+        IBondingCurveBase_v1(fundingManager).buyFor(
+            recipient, firstCollateralIn, 1
+        );
+    }
+
+    function _transferTokenOwnership(
+        ERC20Issuance_v1 issuanceToken,
+        address newAdmin
+    ) private {
+        if (newAdmin == address(0)) {
+            issuanceToken.renounceOwnership();
+        } else {
+            issuanceToken.transferOwnership(newAdmin);
+        }
+    }
+
+    function _transferWorkflowAdminRights(
+        IOrchestrator_v1 orchestrator,
+        address newAdmin
+    ) private {
+        bytes32 adminRole = orchestrator.authorizer().getAdminRole();
+        // if renounced flag is set, add zero address as admin (because workflow must have at least one admin set)
```

Comment:
I understand the comment, but I don't see where the zero address is being set here, as the input `newAdmin` can be non zero. Am I missing something? Comment confuses me :D